CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+
+#### unreleased - upcoming
+
+- Add `requireUserWithAnyRole` and `anyRoleExcept`, to be used in the
```

Comment:
I thought we had ensured that Call Center permissions are always additive, rather than subtractive?